src/serialization/portable/ClassDefinition.ts

```diff
@@ -136,6 +139,9 @@ export class FieldDefinition {
     }
 }
 
+/**
+ * Portable field type.
+ */
 export enum FieldType {
```

Comment:
This is in the public API but it seems that we are not generating documentation for it. Is this intentional?

Reply:
I've already moved this class into a public module in #570, so it shouldn't be a problem.